src/client/java/teammates/client/scripts/DataMigrationForInstructorFeedbackResponseComments.java

```diff
@@ -0,0 +1,56 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+
+import com.googlecode.objectify.Key;
+import com.googlecode.objectify.cmd.Query;
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.storage.entity.FeedbackResponseComment;
+
+/**
+ * Script to set commentGiverType as INSTRUCTOR in all comments by instructor.
+ *
+ * <p>See issue #8830</p>
+ */
+
+public class DataMigrationForInstructorFeedbackResponseComments extends
+        DataMigrationWithCheckpointForEntities<FeedbackResponseComment> {
+
+    public static void main(String[] args) throws IOException {
+        new DataMigrationForInstructorFeedbackResponseComments().doOperationRemotely();
+    }
+
+    @Override
+    protected Query<FeedbackResponseComment> getFilterQuery() {
+        return ofy().load().type(FeedbackResponseComment.class);
+    }
+
+    @Override
+    protected boolean isPreview() {
+        return true;
+    }
+
+    @Override
+    protected String getLastPositionOfCursor() {
+        return "";
+    }
+
+    @Override
+    protected int getCursorInformationPrintCycle() {
+        return 100;
+    }
+
+    @Override
+    protected boolean isMigrationNeeded(Key<FeedbackResponseComment> entity) {
+        FeedbackResponseComment comment = ofy().load().key(entity).now();
+        return comment.getCommentGiverType() == null;
```

Comment:
This won't work as we have a "hack" inside `getCommentGiverType()`

You need to use refelction.

Reply:
Actually, I am curious why
![image](https://user-images.githubusercontent.com/13977235/44956889-5fae0880-aefd-11e8-9814-e797d9cba7c2.png)
is 17 before using reflection.

Reply:
That's because I removed the hack before running script. I didn't know I wasn't supposed to do that. :sweat_smile: